src/ClientModel/SamplingDataPostProcessorTest.cpp

```diff
@@ -0,0 +1,721 @@
+  }
+
+  static const inline std::string kModulePath = "/path/to/module";
+
+  static const inline std::string kFunction1Name = "function1";
+  static constexpr uint64_t kFunction1StartAbsoluteAddress = 0x10;
+  static constexpr uint64_t kFunction1Instruction1AbsoluteAddress = 0x11;
+  static constexpr uint64_t kFunction1Instruction1OffsetInFunction =
+      kFunction1Instruction1AbsoluteAddress - kFunction1StartAbsoluteAddress;
+
+  static const inline std::string kFunction2Name = "function2";
+  static constexpr uint64_t kFunction2StartAbsoluteAddress = 0x20;
+  static constexpr uint64_t kFunction2Instruction1AbsoluteAddress = 0x21;
+  static constexpr uint64_t kFunction2Instruction1OffsetInFunction =
+      kFunction2Instruction1AbsoluteAddress - kFunction2StartAbsoluteAddress;
+
+  static const inline std::string kFunction3Name = "function3";
+  static constexpr uint64_t kFunction3StartAbsoluteAddress = 0x30;
+  static constexpr uint64_t kFunction3Instruction1AbsoluteAddress = 0x31;
+  static constexpr uint64_t kFunction3Instruction1OffsetInFunction =
+      kFunction3Instruction1AbsoluteAddress - kFunction3StartAbsoluteAddress;
+  static constexpr uint64_t kFunction3Instruction2AbsoluteAddress = 0x32;
+  static constexpr uint64_t kFunction3Instruction2OffsetInFunction =
+      kFunction3Instruction2AbsoluteAddress - kFunction3StartAbsoluteAddress;
+
+  static const inline std::string kFunction4Name = "function4";
+  static constexpr uint64_t kFunction4StartAbsoluteAddress = 0x40;
+  static constexpr uint64_t kFunction4Instruction1AbsoluteAddress = 0x41;
+  static constexpr uint64_t kFunction4Instruction1OffsetInFunction =
+      kFunction4Instruction1AbsoluteAddress - kFunction4StartAbsoluteAddress;
+
+  void AddAllAddressInfos() {
+    AddAddressInfo(kModulePath, kFunction1Name, kFunction1Instruction1AbsoluteAddress,
+                   kFunction1Instruction1OffsetInFunction);
+
+    AddAddressInfo(kModulePath, kFunction2Name, kFunction2Instruction1AbsoluteAddress,
+                   kFunction2Instruction1OffsetInFunction);
+
+    AddAddressInfo(kModulePath, kFunction3Name, kFunction3Instruction1AbsoluteAddress,
+                   kFunction3Instruction1OffsetInFunction);
+    AddAddressInfo(kModulePath, kFunction3Name, kFunction3Instruction2AbsoluteAddress,
+                   kFunction3Instruction2OffsetInFunction);
+
+    AddAddressInfo(kModulePath, kFunction4Name, kFunction4Instruction1AbsoluteAddress,
+                   kFunction4Instruction1OffsetInFunction);
+  }
+
+  static constexpr uint64_t kCallstack1Id = 1;
+  static const inline std::vector<uint64_t> kCallstack1Frames{
+      kFunction3Instruction1AbsoluteAddress, kFunction2Instruction1AbsoluteAddress,
+      kFunction1Instruction1AbsoluteAddress};
+  static const inline std::vector<uint64_t> kCallstack1ResolvedFrames{
+      kFunction3StartAbsoluteAddress, kFunction2StartAbsoluteAddress,
+      kFunction1StartAbsoluteAddress};
```

Comment:
Maybe you can add a comment before each of these blocks with the capital letters. Like this: 
`// C B A`

This probably means the letter explanation needs to be moved before this first block

Reply:
Makes sense, I've done something similar, it should be better now.